tests/unit/test___init__.py

```diff
@@ -731,3 +734,59 @@ def test_run_migrate_same_content_local_and_server(caplog, mocked_clients):
 
     assert not returned_migration_reports
     assert any("No community contribution found" in record.message for record in caplog.records)
+    assert not mock_edit_pull_request.called
+
+
+@mock.patch("src.repository.Client.get_pull_request")
```

Comment:
Should we use monkeypatch: https://docs.pytest.org/en/7.1.x/how-to/monkeypatch.html

Reply:
Uhm, I kinda used the one that was used elsewhere in the file to be consistent. It seems that we are using - depending on the file - either monkeypatch or mock.patch, not sure why. But I would keep it consistent per file to start with, but then we should pick one of the two and refactor the others across.

Reply:
yeh, monkeypatch is preferred. No idea where mock.patch came from